### ISSUE TITLE
PM2.5 overprediction by AQMv7 over Colorado

### ISSUE THREAD

**Issue by JianpingHuang-NOAA** [NOAA-EMC/AQM#90]
We are currently seeing PM2.5 over-predictions by AQMv7 over Colorado. This overprediction is not related to wildfire emissions.

![image](https://github.com/NOAA-EMC/AQM/assets/54634403/77610852-00e1-48b8-93db-6d642198516f)

@bbakernoaa

**Comment by bbakernoaa** [comment]
Thanks @JianpingHuang-NOAA.  I noticed this in your UIFWC plots.  There is actually a sign that there is something happening there in the monitor data from your plots.  It may be over predicted.  I've been working on improvements to the inputs for FENGSHA.  We may be able to try these but I'm hesitant to make the change at this point as I don't want to disrupt the implementation.  The sources are not constantly on and so I'm worried about a big change to satisfy a very limited case.

So if you look at the u*/R and compare them to the old ones as shown in the presentation below you can see that the effective dry threshold velocity increases over this region.  So likely the new inputs would decrease the dust to zero or in magnitude over CO.


<img width="1229" alt="image" src="https://github.com/NOAA-EMC/AQM/assets/22104759/ebd036a2-ede3-4ae5-b041-c7eecead9070">

**Comment by bbakernoaa** [comment]
@JianpingHuang-NOAA I have regridded the fengsha inputs to the AQM_NA_13km domain.  You can pick it up here: `/scratch1/RDARCH/rda-arl-gpu/Barry.Baker/emissions/nexus/FENGSHA/Online_CMAQ/FENGSHA_2022_NESDIS_DRAG_ML2_UTHRES_inputs_AQM_NA_13km.nc`

@rmontuoro

**Comment by JianpingHuang-NOAA** [comment]
@Barry Baker - NOAA Federal ***@***.***>  Should I use it to
replace the current Fengsha input
file, FENGSHA_p8_10km_inputs_AQM_NA_13km.nc ?

**Comment by JianpingHuang-NOAA** [comment]
@bbakernoaa Can you answer my previous question ?

**Comment by bbakernoaa** [comment]
Yes you can use it to replace the FENGSHA_p8_10km_inputs_AQM_NA_13km.nc

**Comment by JianpingHuang-NOAA** [comment]
@bbakernoaa 
The updated input file fixed the over-prediction issue over Colorado in July 2023, but it caused a big over-prediction after including it in the rerun  in Sept. 2022.

![plot_grp1 timeseries PM2 5 2022-09-01_12 2022-09-30_12 all CONUS](https://github.com/NOAA-EMC/AQM/assets/54634403/eb94b8f6-665d-499c-991e-c3fbd1d824ce)

I have to change the input file to the original one.

**Comment by JianpingHuang-NOAA** [comment]
![plot_grp4 spatial_overlay PM2 5 2022-09-01_12 2022-09-30_12 all CONUS airnow_c83](https://github.com/NOAA-EMC/AQM/assets/54634403/44dc968a-54c2-4f52-a6f6-e7a281cea478)

**Comment by bbakernoaa** [comment]
Can you point me to the output because this isn't helping much....? it actually looks like it fixed the issue over CO.

**Comment by bbakernoaa** [comment]
Closing and will reopen a new issue for AQMv8 if continued issues persist.
